src/Security/Member.php

```diff
@@ -114,7 +114,7 @@ class Member extends DataObject
      * statement.
      *
      * @var array
-     * @todo Generic implementation of $searchable_fields on DataObject,
+     * Generic implementation of $searchable_fields on DataObject,
      * with definition for different searching algorithms
      * (LIKE, FULLTEXT) and default FormFields to construct a searchform.
```

Comment:
```suggestion
```
Please always remove the _full_ todo comment.

Reply:
Updated